packages/flutter_tools/test/integration.shard/test_driver.dart

```diff
@@ -679,8 +679,7 @@ class FlutterTestTestDriver extends FlutterTestDriver {
       'test',
       '--disable-service-auth-codes',
       '--machine',
-      '-d',
-      'flutter-tester',
```

Comment:
Conceptually we are still launching flutter_tester for these tests, right? Who is going to terminate it?

Reply:
Previously we were launching two.  Now we launch one.  Testing locally on Windows, this doesn't leak the flutter_tester.exe process we used to leak.